src/unix/mod.rs

```diff
@@ -66,10 +66,21 @@ s! {
     // See https://sourceware.org/bugzilla/show_bug.cgi?id=16437
     pub struct timespec {
```

Comment:
This struct has some complicated config. Maybe it would be easiest to split? Config the current one non-gnu and move the rest to linux/gnu.

Reply:
Done